src/Source/PotreeSource.js

```diff
@@ -86,6 +87,11 @@ class PotreeSource extends Source {
                 return cloud;
             });
     }
+
+    onLayerAdded(layer) {
+        super.onLayerAdded(layer);
+        layer.out.extent = Extent.fromBox3(layer.out.crs, layer.out.root.bbox);
+    }
```

Comment:
would this layer extent initialization not also be useful for the other EntwinePointTileSource and C3DTilesSource?

Reply:
May be remove this calculation?